### ISSUE TITLE
Error when adding a button in version 57.3

### ISSUE THREAD

**Issue by dilanSachi** [facebook/react-native#21883]
java.lang.String cannot be cast to com.facebook.react.uimanager.AccessibilityDelegateUtil$AccessibilityRole
